ansible/awx
# Adding capability of pretty error pages

##### SUMMARY

AWX will present the default nginx error pages for 502/504 errors. This PR allows us to add files in the static directory like `custom_XXX.html` and will instruct nginx to respond with an html page on error.

To test:
* start AWX and load the login page
* Shell into the container and run `supervisorctl stop tower-processes:awx-uwsgi`
* Reload the web page, you should see the custom 504 page
* Shell into the container and run `supervisorctl start tower-processes:awx-uwsgi`
* The page should eventually reload back into the login page. (sometimes depending on the refresh rate you might see a 404 here).



##### ISSUE TYPE

 - Bug, Docs Fix or other nominal change

##### COMPONENT NAME

 - Other

##### AWX VERSION

```

```


##### ADDITIONAL INFORMATION



```

```
